examples/test/test_resnet_vitis_ai_ptq_cpu.py

```diff
@@ -30,6 +32,10 @@ def setup():
 @pytest.mark.parametrize("execution_order", ["pass-by-pass"])
 @pytest.mark.parametrize("system", ["local_system", "aml_system"])
 @pytest.mark.parametrize("olive_json", ["resnet_vitis_ai_ptq_cpu.json"])
+@pytest.mark.skipif(
```

Comment:
Is this a different issue from the quantization preprocessing issue?

Reply:
yes, the Vitis code need to be updated since the calibrate API is changed. I offline synced with AMD guys.